recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
-{% set version = "18.18.2" %}
+{% set version = "20.9.0" %}
```

Comment:
The latest 20 version is 20.17.0: https://github.com/nodejs/node/tree/v20.17.0

Reply:
Done

Reply:
I added adjustments to solve https://github.com/nodejs/node/issues/52223.
And updated the `install.py` command: `$PYTHON tools/install.py install --dest-dir ${PREFIX} --prefix`